package.json

```diff
@@ -9,11 +9,12 @@
   },
   "dependencies": {
     "babel-eslint": "4.1.8",
-    "eslint": "codeclimate/eslint.git#d24d0b",
-    "eslint-config-airbnb": "^5.0.0",
+    "eslint": "^2.2.0",
```

Comment:
Agreed - makes sense to lock this to a specific version, since the image will be locked anyway! Makes it easier to be sure of what's on the image too.